react-main/src/redux/login/reducer.js

```diff
@@ -0,0 +1,36 @@
+const initialState = {
+  message: '',
+  session: false
+};
+
+export const loginReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'LOGIN_USER':
+      return {
+        ...state
+      };
+    case 'LOGIN_SUCCESS':
+      return {
+        ...state,
+        message: action.message
```

Comment:
Action usually contains a type and a payload. Inside the payload goes all the data, like message and session